packages/peregrine/lib/talons/Wishlist/GalleryButton/helpers/useSingleWishlist.js

```diff
@@ -0,0 +1,130 @@
+import { useCallback, useMemo, useState } from 'react';
```

Comment:
Just a note: This pattern gels well with the refactor described in #3141 with some refactoring.

Basically in that refactor, there is no single and multiple difference. The difference is only CE and EE and the logic single and multiple in case EE is contained in that talon itself.